trunk/SUAVE/Methods/Performance/electric_V_h_diagram.py

```diff
@@ -129,18 +130,18 @@ def mini_mission(altitude, speed):
                 D = -results.segments.single_point.conditions.frames.wind.drag_force_vector[0][0]
 
                 # Determine Propeller Power at Altitude and Speed
-
-                P = propeller_single_point(vehicle.networks.battery_propeller,
-                                           analyses,
+                prop_key = list(vehicle.networks.battery_propeller.propellers.keys())[0]
```

Comment:
Wait, this won't work as intended anymore. This was supposed to run a vehicle not just a propeller. The V-h diagrams are nonsense if it's just a single prop.

Reply:
propeller_single_point() only used the provided or first indexed propeller in the network previously. No functionality has changed with this PR, so if this is supposed to do something different then it wasn't doing it before. We can address this issue in a new PR.

Reply:
Oh jeez, we need to make an issue then.